Corona-Warn-App/src/androidTest/java/testhelpers/BaseUITest.kt

```diff
@@ -13,4 +15,11 @@ abstract class BaseUITest : BaseTest() {
     fun clearAllViewModels() {
         MockViewModelModule.CREATORS.clear()
     }
+
+    inline fun <reified F: Fragment>captureScreenshot(suffix:String = "") {
+        val name = F::class.simpleName
+        launchFragmentInContainer2<F>()
+        Thread.sleep(SCREENSHOT_DELAY_TIME)
+        Screengrab.screenshot(name.plus(suffix))
+    }
```

Comment:
👍 , Please add it in `TestExtensions` file please consider also that in some situations , a different theme ID has to be passed to it. ex : captureScreenshot<Fragment>(R.style.AppTheme_Diary)